dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hdfs/HdfsPermissionExtractionTask.java

```diff
@@ -77,14 +78,17 @@ private void doRun(Writer output) throws IOException {
     conf.set("fs.defaultFS", "hdfs://" + clusterHost + ":" + port + "/");
 
     try (FileSystem fs = FileSystem.get(conf);
-        ScanContext scanCtx = new ScanContext(fs, output)) {
+        ScanContext scanCtx = new ScanContext(fs, output);
+        ExecutorManager execManager =
+            new ExecutorManager(
+                ExecutorManager.newExecutorServiceWithBackpressure(
+                    "hdfs permission extraction", poolSize))) {
```

Comment:
Nit: conventionally this name is hyphen-separated

Reply:
Done.